shoop/admin/modules/orders/json_order_creator.py

```diff
@@ -254,6 +247,8 @@ def create_source_from_state(self, state, creator=None):
         :type state: dict
         :param creator: Creator user
         :type creator: django.contrib.auth.models.User|None
+        :param save: Flag whether order customer or addresses is saved to db
```

Comment:
You can afford to spell "database" out :) Also, shouldn't it be customer _and_ addresses?